packages/shared-influx-pieces/src/lib/functions.ts

```diff
@@ -0,0 +1,22 @@
+import { envIsNullish } from '@skyra/env-utilities';
+import { container } from '@skyra/http-framework';
+import { Client, areInfluxCredentialsSet, getInfluxConnectionOptions } from '@skyra/influx-utilities';
+
+let interactionCount: number;
+let initialized = false;
+
+export function isInfluxInitialized() {
+	return initialized;
+}
+
+export function initializeInflux() {
+	if (envIsNullish('INFLUX_ENABLED') || !areInfluxCredentialsSet()) return;
```

Comment:
This should be `!envParseBoolean('INFLUX_ENABLED', true)` since this flag is actually opt-out. It being required would be counter-intuitive.